backend/cypress_test.ini

```diff
@@ -209,7 +209,7 @@ email.template_dir = %(here)s/tracim_backend/templates/mail
 # processing_mode may be sync or async,
 # if async is choosen, you need also to run mail_notifier daemon
 # see README for more info.
-email.processing_mode = sync
+jobs.processing_mode = sync
 ## with async, please also configure redis below
 ; email.async.redis.host = localhost
```

Comment:
change these commented value too for consistency (NB: cypress_test.ini should probably be deleted or simplify later)

Reply:
Changed.